turbopack/crates/turbo-tasks-backend/src/backend/operation/aggregation_update.rs

```diff
@@ -180,51 +192,48 @@ impl AggregatedDataUpdate {
         if let Some((dirty_container_id, count)) = dirty_container_update {
             // When a dirty container count is increased and the task is considered as active
             // `AggregateRoot` we need to schedule the dirty tasks in the new dirty container
-            if *count > 0 && task.has_key(&CachedDataItemKey::AggregateRoot {}) {
+            let current_session_update = count.get(session_id);
+            if current_session_update > 0 && task.has_key(&CachedDataItemKey::AggregateRoot {}) {
                 queue.push(AggregationUpdateJob::FindAndScheduleDirty {
                     task_ids: vec![*dirty_container_id],
                 })
             }
-            let mut added = false;
-            let mut removed = false;
+
+            let mut aggregated_update = Default::default();
```

Comment:
Nit: prefer the more explicit `DirtyContainerCount::default()` over `Default::default()` to make the type more obvious.

```suggestion
            let mut aggregated_update = DirtyContainerCount::default();
```